app/controllers/vm_common.rb

```diff
@@ -1677,6 +1677,39 @@ def set_form_vars
     session[:edit] = @edit
   end
 
+  ACTION_TYPES = {
+    "advanced_settings"   => ->(amount) { n_("Advanced Setting", "Advanced Settings", amount) },
```

Comment:
Hmmm...lamdbas at the const scope concern me a bit if the binding they capture is too wide.

@matthewd Can you review this PR?  I'm hoping if we can get a solid recommendation to @mzazrivec, then he can just knock out this pattern for getting rid of the dictionary.